modules/usbip.nix

```diff
@@ -0,0 +1,58 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  usbipd-win-auto-attach = pkgs.fetchurl {
+    url = "https://raw.githubusercontent.com/dorssel/usbipd-win/9e259e74e225ed9321484366a79545b00a18185c/Usbipd/wsl-scripts/auto-attach.sh";
```

Comment:
Can we use the release tags?

Reply:
A release have not happened, so I can use the release tag.